<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CALENDARIO</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>331</width>
     <height>191</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt</string>
   </property>
   <property name="text">
    <string>Ingrese la fecha de última aplicación del fertilizante:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>281</y>
     <width>311</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelFecha">
      <property name="styleSheet">
       <string notr="true">font: 14pt</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt;</string>
      </property>
      <property name="text">
       <string>Seleccionar fecha</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
